src/test/java/com/onwelo/tinvalidator/validator/GBValidatorTest.java

```diff
@@ -0,0 +1,75 @@
+package com.onwelo.tinvalidator.validator;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+public class GBValidatorTest {
+
+    @ParameterizedTest
+    @DisplayName("contain five, nine or twelve characters")
+    @CsvSource(value = {
+            "GD111, true",
+            "GD555, false",
+            "HA666, true",
+            "HA333, false",
```

Comment:
it has five characters but is false, I know that it isn't apply to regex but with name of the test someone can get confused